client/src/pages/formatFile.js

```diff
@@ -27,12 +34,73 @@ const formatFile = () => {
   if (!data) {
     return <div>Loading...</div>;
   }
+
+  // change message hook
+  //   wait for the hook to change
+  // send the message to the data to create own pastebin
+  // send back because of the promiseHooks(await)
+  // change url
+  const changeData = async (message) => {
+    try {
+      const res = await fetch("http://localhost:8000/store_paste", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ text: message }),
+      });
+
+      if (!res) {
+        throw new Error("No response saved");
+      }
+
+      const msg = await res.json();
+      return msg.name;
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  // how about instead /remix/:name renders a new component RemixPaste instead
+  // RemixPaste checks the id when the page loads
+  //and renders a textarea with the initial value of whatever the paste is
+
+  // when the Remix Button is pressed
+  // Navigates to the new component remixPaste.js
+  // once it gets to the page, the text area is already open with the previous message inside
+  // when the buttone is pressed the page renders to the new page ../{name}
+
+  const handleClick = async (message, title) => {
```

Comment:
we can remove this? this and the other comment about the unused hook also goes for remixPaste.js

Reply:
resolved, ty